src/colvar/RMSD.cpp

```diff
@@ -117,11 +117,13 @@ you are working with natural units.  If you are working with natural units then
 should be in your natural length unit.  You can find more details on the PDB file format [here](http://www.wwpdb.org/docs.html).
 Please make sure your PDB file is correctly formatted.  More detail on the format for PDB files can be found in the documentation for the [PDB2CONSTANT](PDB2CONSTANT.md) action.
 
-The following input uses a different method to calculate the RMSD distance as you can tell from the TYPE=OPTIMAL on the input line.
+The following input uses a different method to calculate the RMSD distance as you can tell from the TYPE=OPTIMAL on the input line.  In addition, because we have added the
+SQURED flag on the input line, we are calculating $d(X,X')^2$ rather than $d(X,X')$.  Calculating $d(X,X')^2$ is less computationally expensive than computing $d(X,X')$ as
+you avoid a square root operation.
```

Comment:
@gtribello in practice I think the extra cost is negligible (after you have done an MSD calculation), I think this is more for numerical stability